crates/biome_css_analyze/src/utils.rs

```diff
@@ -127,3 +130,21 @@ pub fn is_pseudo_elements(prop: &str) -> bool {
         || SHADOW_TREE_PSEUDO_ELEMENTS.contains(&prop)
         || OTHER_PSEUDO_ELEMENTS.contains(&prop)
 }
+
+pub fn is_kown_properties(prop: &str) -> bool {
+    KNOWN_PROPERTIES.contains(&prop)
+        || KNOWN_CHROME_PROPERTIES.contains(&prop)
+        || KNOWN_EDGE_PROPERTIES.contains(&prop)
+        || KNOWN_EXPLOLER_PROPERTIES.contains(&prop)
+        || KNOWN_FIREFOX_PROPERTIES.contains(&prop)
+        || KNOWN_SAFARI_PROPERTIES.contains(&prop)
+        || KNOWN_SUMSUNG_INTERNET_PROPERTIES.contains(&prop)
+        || KNOWN_US_BROWSER_PROPERTIES.contains(&prop)
```

Comment:
Thank you for this massive effort. To make things as fast as possible, we want to use `.binary_search` instead of `.contains`. This means that these new lists **must be sorted** and to avoid regressions we must have tests: 

https://github.com/biomejs/biome/blob/01c440d78343a1217271f874c64c05ffcb725077/crates/biome_js_analyze/src/globals/module/node.rs#L175-L179

Can you address that?

Reply:
@ematipico 
Thank you always. Yes, I will do that tomorrow:)